src/main/java/ch/njol/skript/expressions/ExprDropsOfBlock.java

```diff
@@ -0,0 +1,106 @@
+/**
+ * This file is part of Skript.
+ *
+ * Skript is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Skript is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.expressions;
+
+import org.bukkit.block.Block;
+import org.bukkit.entity.Entity;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.ItemStack;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.skript.log.ErrorQuality;
+import ch.njol.util.Kleenean;
+
+@Name("Drops Of Block")
+@Description("A list of the items that will drop when a block is broken. Note that 'as %entity%' only works on minecraft 1.15+.")
+@Examples({"on break of block:",
+	"\tgive drops of block using player's tool to player"})
+@Since("INSERT VERSION")
+public class ExprDropsOfBlock extends SimpleExpression<ItemStack> {
+	
+	static {
+		Skript.registerExpression(ExprDropsOfBlock.class, ItemStack.class, ExpressionType.COMBINED, "drops of %block% [(using|with) %-itemstack% [(1¦as %-entity%)]]", "%block%'s drops [(using|with) %-itemstack% [(1¦as %-entity%)]]");
```

Comment:
use ItemType, not ItemStack. Also, it should support multiple blocks

Reply:
Why should I do that?

Reply:
It's rather complex to explain here but basically, Skript only passes around itemtypes, not itemstacks and even if it may make this specific case a bit more complex, it's better for expressions to encourage the usage of ItemType so that addon developers don't start passing ItemStacks around and generating weird behaviour in some expressions.

Reply:
👍 okay I'll add that afterwards, thanks

Reply:
You could also split this into multiple lines

Reply:
How would I implement it?
Would I do something along the lines of
```java
				List<ItemStack> stacks = (List<ItemStack>) block.getDrops(item.getRandom());
				int size = stacks.size();
				ItemType[] result = new ItemType[size];
				for (int i = 0; i < size; i++) {
					result[i] = new ItemType(stacks.get(i));
				}
				return result;
```

Reply:
Yeah, that should do